services/admin/serverless.yml

```diff
@@ -36,6 +36,7 @@ provider:
             - dynamodb:UpdateItem
             - dynamodb:DeleteItem
           Resource:
+            - arn:aws:dynamodb:*:*:table/product-change-request-2
```

Comment:
This resource does not exist in the production account... which I was worried about until I noticed it does not exist in the master account (VAL) either, so probably not important.